addOns/soap/src/test/java/org/zaproxy/zap/extension/soap/PassiveScannerTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2016 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.soap;
+
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+import org.zaproxy.zap.testutils.PassiveScannerTestUtils;
+
+abstract class PassiveScannerTest<T extends PluginPassiveScanner>
```

Comment:
Not really needed just for one class (IMO).

Reply:
How else do I deal with the tests then?

Reply:
It just need to move the override to the existing class (and extend the same this one is extending).